nexus/tests/integration_tests/console_api.rs

```diff
@@ -335,33 +335,34 @@ async fn test_session_me(cptestctx: &ControlPlaneTestContext) {
         .execute()
         .await
         .expect("failed to get current user")
-        .parsed_body::<views::User>()
+        .parsed_body::<views::SessionMe>()
         .unwrap();
 
     assert_eq!(
         priv_user,
-        views::User {
+        views::SessionMe {
             id: USER_TEST_PRIVILEGED.id(),
             display_name: USER_TEST_PRIVILEGED.external_id.clone(),
             silo_id: DEFAULT_SILO.id(),
+            group_ids: vec![],
         }
     );
 
-    // make sure it returns different things for different users
     let unpriv_user = NexusRequest::object_get(testctx, "/session/me")
         .authn_as(AuthnMode::UnprivilegedUser)
         .execute()
         .await
         .expect("failed to get current user")
-        .parsed_body::<views::User>()
+        .parsed_body::<views::SessionMe>()
         .unwrap();
 
     assert_eq!(
         unpriv_user,
-        views::User {
+        views::SessionMe {
             id: USER_TEST_UNPRIVILEGED.id(),
             display_name: USER_TEST_UNPRIVILEGED.external_id.clone(),
             silo_id: DEFAULT_SILO.id(),
+            group_ids: vec![],
```

Comment:
It would be cool to be able to add the user to a group, then refetch and see the group here. But I see that there is no endpoint for adding a user to a group and modification of group memberships only happens during JIT user create. A next-best thing would be if the built-in privileged and unprivileged users were in groups, so at least this vec would come back non-empty. @jmpesp any thoughts?

Reply:
In `nexus/tests/integration_tests/saml.rs`, `test_correct_saml_response_with_group_attributes` tests that groups are JITed correctly, and `test_post_saml_response` calls `/session/me` to validate a successful SAML login. I'm not sure about the built-in users being part of some fake groups when a test(s) could be done there.

Reply:
I see. In 0e61d09a3fd528892fb9ee60021098a47c2acdbb I modified `test_post_saml_response` to include groups and assert that the groups are there on `/session/me`. One downside is that test didn't used to be about groups and now it is. Not sure if that is really a problem.

Reply:
I think it's not a problem, though if the order of groups returned changes then that `assert_eq` will fire.

Reply:
Add a cute helper for that in 52b9f46